cmd/up.go

```diff
@@ -132,11 +133,12 @@ func Up() *cobra.Command {
 	cmd.Flags().IntVarP(&remote, "remote", "r", 0, "configures remote execution on the specified port")
 	cmd.Flags().BoolVarP(&autoDeploy, "deploy", "d", false, "create deployment when it doesn't exist in a namespace")
 	cmd.Flags().BoolVarP(&forcePull, "pull", "", false, "force dev image pull")
+	cmd.Flags().BoolVarP(&resetSyncthing, "reset-syncthing", "", false, "reset the syncthing database")
```

Comment:
I added this parameter to make it easier to reset the dev environment if syncthing database is corrupted